DataLink.tex

```diff
@@ -66,16 +66,18 @@ \section*{Conformance-related definitions}
 \section{Introduction}
 
 This specification defines mechanisms for connecting metadata about
-discovered datasets to the data, related data products, and web services
-that can act upon the data.
+discovered VO items to  related data products, and web services
```

Comment:
I agree that we need to broaden beyond datasets, but "VO items" is odd. Maybe instead of "metadata about discovered VO items to..." we should say "metadata discovered via one service to...".
Plus, the comma after products should be removed.